.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature request
+assignees: ''
+
+---
+
+<!--
+Thank you for contributing to the Firebase community!
+
+Have a feature request?
+========================
+Great, we love hearing how we can improve our products! However, GitHub is not the place to submit them. Please submit your feature requests to:
+https://firebase.google.com/support/contact/bugs-features/
+
+Have a usage question?
+=======================
+We get lots of those and we love helping you, but GitHub is not the best place for them and they
+will be closed. Here are some resources to get help:
+
+- Start with the quickstart: https://firebase.google.com/docs/functions/write-firebase-functions
+- Go through the guide: https://firebase.google.com/docs/functions/
+- Read the full API reference: https://firebase.google.com/docs/reference/functions/
+- Browse some examples: https://github.com/firebase/functions-samples
+
+If the official documentation doesn't help, try asking through our official support channels:
+
+https://firebase.google.com/support/
+
+*Please avoid double posting across multiple channels!*
+
+Think you found a bug?
```

Comment:
Is this too repetitive? I think also it would be hard for the dev to go back to the selection step. Might be less confusing to just not include this here.

Reply:
Let's just put a one liner here as discussed: "Think this is a bug? Don't use this form."

Reply:
I've updated this phrasing. :)